api/src/main/java/org/openmrs/module/fhir/api/db/hibernate/HibernateFHIRDAO.java

```diff
@@ -15,9 +15,21 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.hibernate.Criteria;
 import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Restrictions;
+import org.hibernate.jdbc.Work;
+import org.openmrs.Obs;
```

Comment:
Criteria, Restrictions, and Obs imported but never used?

Reply:
@corneliouzbett Thanks for the catch.